onnxruntime/test/onnx/runner.cc

```diff
@@ -459,6 +497,12 @@ void SeqTestRunner::Start(ORT_CALLBACK_INSTANCE pci, size_t) {
 }
 
 void RunSingleTestCase(ITestCase* info, Ort::Env& env, const Ort::SessionOptions& sf, size_t concurrent_runs, size_t repeat_count, PThreadPool tpool, ORT_CALLBACK_INSTANCE pci, TestCaseCallBack on_finished) {
+
+  // for test in immutable list, do not even run it
+  if (immutable_broken_tests.find(info->GetTestCaseName()) != immutable_broken_tests.end()) {
```

Comment:
What if people the this tool out of CI ? This piece of code will create surprise to them.

Reply:
Suggestions?

Reply:
At the very least it should be properly documented, saying why we do not run them. Perhaps, both here is and at the place the the list is defined.

---
In reply to: [291396237](https://github.com/microsoft/onnxruntime/pull/1181#discussion_r291396237) [](ancestors = 291396237)

Reply:
Added comments.

Reply:
Commented.

Reply:
Please, describe why we do not run them and why the list should not be changed. (Because we do not expect any more test exclusions for that reason).

---
In reply to: [291785248](https://github.com/microsoft/onnxruntime/pull/1181#discussion_r291785248) [](ancestors = 291785248)